src/h5web/providers/mock/data.ts

```diff
@@ -153,6 +153,38 @@ export function makeNxDataGroup(
   );
 }
 
+export function makeNxEntry(
```

Comment:
I like this naming convention. Mind renaming `makeNxDataGroup` to `makeNxData` ?

Reply:
Alternatively, we could also have a generic `makeNxGroup` that accepts a class and a record of IDs, which would be called by `makeNxData` and perhaps `makeNxGroupWithDefault`. I'm fine with the current implementation, though ; just a thought.

Reply:
Meh, not sure the abstraction is worth it